Cargo.toml

```diff
@@ -13,6 +13,9 @@ repository = "https://github.com/auxoncorp/ekotrace"
 readme = "README.md"
 build = "build.rs"
 
+[features]
```

Comment:
I think we should explicitly add a `default = []` line here to make it quite clear the that the use of std is bonus rather than baseline.